app/renderer/actions/Inspector.js

```diff
@@ -510,6 +512,107 @@ export function setLocatorTestElement (elementId) {
   };
 }
 
+/**
+ * Given an element ID found through search, and its bounds,
+ * attempt to find and select this element in the source tree
+ */
+export function selectLocatedElement (source, bounds, id) {
+  const UPPER_FILTER_LIMIT = 10;
+
+  // Parse the source tree and find all nodes whose bounds match the expected bounds
+  // Return the path + xpath of each node
+  function findPathsMatchingBounds () {
+    if (bounds && source.children && source.children[0].attributes) {
```

Comment:
I would rather flip this condition and make it to a guard.
Guards are easier to read than deeply nested ifs

Reply:
Fixed